File: Asocioacion/php/tercerEjercicio/Piloto.php
<?php

class Piloto{

    private string $nombrePiloto;
    private $aeronave = [];
    private $vuelo = [];

    public function __construct(string $nombrePiloto){

        $this->nombrePiloto = $nombrePiloto;

    }

    //Nombre del Piloto
    public function getNombrePiloto(){
        return $this->nombrePiloto;
    }
    public function setNombrePiloto($nombrePiloto){
        $this->nombrePiloto = $nombrePiloto;
    }

    //Agregar y visualizar vuelos
    public function agregarVuelos($vuelos){
        array_push($this->vuelo,$vuelos);
    }
    public function verVuelos(){
        return $this->vuelo;
    }

    //Agregar y visualizar 
    public function agregarAeronaves($aeronaves){
        array_push($this->aeronave,$aeronaves);
    }
    public function verAeronaves(){
        return $this->aeronave;
    }

}

?>

File: Asocioacion/python/tercerEjercicio/Piloto.py
class Piloto:
    
    c = 0
    
    def __init__(self, nombrePiloto):
        self.nombrePiloto = nombrePiloto
        self.vuelo = []
        self.aeronave = []
    
    #Nombre del piloto    
    
    def getNombrePiloto(self):
        return self.nombrePiloto
    
    def setNombrePiloto(self, nombrePiloto):
        self.nombrePiloto = nombrePiloto
        
    #Metodo de agregacion y visualizacion de vuelos
    
    def agregarVuelo(self, vuelo):
        self.vuelo.append(vuelo)
        
    def verVuelo(self):
        return self.vuelo
    
    #Metodo de agregacion y visualizacion de aeronaves
    
    def agregarAeronave(self,aeronave):
        self.aeronave.append(aeronave)
        
    def verAeronave(self):
        return self.aeronave